multiversx_sdk/core/transactions_factories/smart_contract_transactions_factory.py

```diff
@@ -88,26 +88,33 @@ def create_transaction_for_execute(self,
                                        arguments: Sequence[Any] = [],
                                        native_transfer_amount: int = 0,
                                        token_transfers: Sequence[ITokenTransfer] = []) -> Transaction:
-        number_of_tokens = len(token_transfers)
+        transfers = deepcopy(token_transfers)
+        transfers = list(transfers)
```

Comment:
This is sufficient: `transfers = list(token_transfers)` - since it creates a new list, with same elements. Deep copy is not necessary.

Reply:
indeed, removed it